modules/300-prometheus/monitoring/prometheus-rules/prometheus-storage.yaml

```diff
@@ -0,0 +1,52 @@
+- name: d8.prometheus.storage
+  rules:
+  - alert: PrometheusDiskUsage
+    expr: |
+      d8_prometheus_fs_used_percent > 95
```

Comment:
I am afraid of the number 95. Why have we picked it?

Reply:
This alert is just in case something went wrong and the retain didn't work.